docs/content/docs/install/manual_demo/_index.md

```diff
@@ -71,10 +71,10 @@ osm namespace add bookstore bookbuyer bookthief bookwarehouse
 Deploy `bookbuyer`, `bookthief`, `bookstore`, `bookwarehouse` applications:
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookbuyer.yaml
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookthief.yaml
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookstore.yaml
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/main/docs/example/manifests/apps/bookwarehouse.yaml
+kubectl apply -f docs/example/manifests/apps/bookbuyer.yaml
+kubectl apply -f docs/example/manifests/apps/bookthief.yaml
+kubectl apply -f docs/example/manifests/apps/bookstore.yaml
+kubectl apply -f docs/example/manifests/apps/bookwarehouse.yaml
```

Comment:
These links were updated to absolute URLs so the user wouldn't need to check out the repo to run the demo. 
PR https://github.com/openservicemesh/osm/pull/2733 updated it for this purpose. /cc @michelleN

Reply:
I see. We do have this in the `Configure Prerequisites` section of this page right now
```
- Have a local clone of the OSM GitHub Repo
  - `git clone https://github.com/openservicemesh/osm.git`
  - `cd osm`
 ```

Reply:
Cloning the repo should be optional if one uses released binaries available directly on the release page.